package.json

```diff
@@ -1,56 +1,36 @@
 {
-  "author": "Lloyd Hilaiel <lloyd@hilaiel.com> (http://lloyd.io)",
-  "name": "convict",
-  "description": "Featureful configuration management library for Node.js (nested structure, schema validation, etc.)",
-  "keywords": [
-    "configuration",
-    "config",
-    "key value store",
-    "schema",
-    "nested",
-    "validation"
-  ],
-  "version": "5.2.0",
-  "license": "Apache-2.0",
-  "homepage": "https://github.com/mozilla/node-convict",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mozilla/node-convict.git"
-  },
-  "bugs": "https://github.com/mozilla/node-convict/issues",
-  "engines": {
-    "node": ">=6"
-  },
-  "scripts": {
-    "posttest": "npm run lint",
-    "test": "mocha --check-leaks test/*-tests.js",
-    "posttest:coverage": "npm run lint",
-    "test:coverage": "nyc --nycrc-path=nycrc.json --cache-dir=/tmp/nyc_cache_convict mocha test/*-tests.js",
-    "pretest:ci": "npm run lint",
-    "test:ci": "npm run test:coverage && cat /tmp/nyc_report_convict/lcov.info | coveralls",
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
-    "version": "./assert_changelog_ready $npm_package_version"
-  },
-  "files": [
-    "lib"
-  ],
-  "main": "lib/convict.js",
+  "name": "node-convict",
+  "private": true,
   "dependencies": {
+    "json5": "2.1.0",
```

Comment:
This `json5` should not be present anymore as a `dependency`

Reply:
Now, json is devDepencies.

ok